bigchaindb/pipelines/election.py

```diff
@@ -0,0 +1,51 @@
+import rethinkdb as r
```

Comment:
Can you add a module-level docstring to explain in few words what this module is about? Just a short intro.